js/_diff.js

```diff
@@ -279,16 +279,20 @@ function getDiffLines(hole, left, right, pos, argv, isArgDiff) {
     const padding = isUnchanged ? 3 : 20;
     const skipMiddle = numLines > 2 * padding + 1;
     for (let i=0; i<numLines; i++) {
-        if (skipMiddle && padding <= i && i < numLines - padding) continue;
-
-        const row = <tr/>;
+        const row = <tr />;
         rows.push(row);
 
-        if (skipMiddle && i === padding) {
-            row.append(<td class="diff-skip" colspan={isArgDiff ? 3 : 4}>
-                {`@@ ${numLines - 2 * padding} lines omitted @@`}
-            </td>);
-            continue;
+        if (skipMiddle) {
+            if (padding < i && i < numLines - padding) {
+                // In the middle; skip the line
+                continue;
+            } else if (i === padding) {
```

Comment:
Move the else if down a line to fix the lint violation but I'd actually just make it an if since the continue doesn't return.

With a little whitespace rearrangement we can do something like:
```js
if (skipMiddle) {
    // In the middle; skip the line.
    if (padding < i && i < numLines - padding) continue;

    // At the start of the middle; add a line saying lines omitted.
    if (i === padding) {
        row.append(<td class="diff-skip" colspan={isArgDiff ? 3 : 4}>
            {`@@ ${numLines - 2 * padding} lines omitted @@`}
        </td>);
        continue;
    }
}
```